src/App.css

```diff
@@ -32,6 +40,34 @@
   color: #61dafb;
 }
 
+.column-container {
```

Comment:
As we discussed yesterday, we aim to "put things that are related close to each other". That is, we prefer the css rules for columns to be in a file close to the column component, same for the card. etc. 

It's fine for now, just let's think about it for future developments

Reply:
Yes of course. I did it as a temporary solution until we decide how we want to change the appearance or better to say by use of which approach as we discussed.

Reply:
I explained to other members, how this approach works but I didn't change this PR. Because they are working on the appearance.

Reply:
me and Morvarid have found a way in CSS to list the 8 columns inside one page and to keep the position of their titles fixed. We could start to create a new branch on this but we are wandering wheter is better to do the git pull origin from this task or wait that it is merged to the master.

Reply:
Great, all the CSS are temporary and I just did them for showcase so you can do whatever you want and if you need any help let me know. :grin: